src/viam/sdk/components/camera/camera.cpp

```diff
@@ -24,6 +25,96 @@ API API::traits<Camera>::api() {
     return {kRDK, kComponent, "camera"};
 }
 
+// Appends a uint64_t value in big-endian format to a byte vector and updates the offset.
+void append_uint64_big_endian(std::vector<unsigned char>& data, size_t& offset, uint64_t value) {
```

Comment:
Utility functions like this should be in the unnamed namespace so they don't have linkage.

Reply:
Thanks for the review! New to C++ so learning a lot :D